android-interop-testing/build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
     implementation (libraries.google_auth_oauth2_http) {
         exclude group: 'org.apache.httpcomponents'
     }
+    censusGrpcMetricDependency 'implementation'
```

Comment:
Would this be `runtimeOnly`?

Reply:
No, `io.opencensus.contrib.grpc.metrics.RpcMeasureConstants` is used in `src/main/java/io/grpc/testing/integration/AbstractInteropTest.java`, which is in its sourceSets.

Reply:
`grpc-testing` itself depends on census metrics already. `android-interop-testing` (including the included `grpc-interop-testing` sources) does not have any direct API interaction with with census right? So it should not have a direct dependency on it.

Reply:
It needs it to build.  Without it the build would fail. `grpc-testing` itself depends on census metrics already, that's true, but here we use `implementation project(':grpc-testing')`, so all `grpc-testing`'s dep is in runtimeClasspath, not in compileClasspath. We need metrics in complieClasspath to compile the sourceSet.

Reply:
I corrected and updated my reply.